src/plugins/intel_gpu/src/graph/impls/ocl/paged_attention.cpp

```diff
@@ -1006,18 +1033,43 @@ struct paged_attention_impl : multi_stage_primitive<paged_attention> {
             kernels_data.push_back(kv_cache_rotate_kernel_selector.get_best_kernel(kv_cache_rotate_kernel_params));
         }
 
+        auto is_sdpa_micro = [](const kernel_selector::kernel_data& kd) -> bool {
+            OPENVINO_ASSERT(kd.kernels.size() != 0);
+            return !kd.kernels[0].micro_kernels.empty();
+        };
+
+        // If sdpa_micro kernel is supported, create both sdpa and sdpa_micro kernels
+        // And add sdpa_micro kernel to the end of the kernels_data vector
+        if (sdpa_kernel_params.conf.paged_attention_sliding_window == 0 && is_sdpa_micro(kernels_data[Stage::SDPA])) {
+            sdpa_kernel_params.conf.paged_attention_sliding_window = std::numeric_limits<size_t>::max();
```

Comment:
Can't it be just with "sdpa_kernel_params.should_use_sdpa_opt = true;" ?

Reply:
Fixed